vernac/mltop.ml

```diff
@@ -94,6 +94,9 @@ module PluginSpec : sig
      should strengthen this invariant. *)
   val digest : t -> Digest.t list
 
+  (** bool = true for implicit dependencies *)
+  val add_deps : t list -> (bool * t) list
```

Comment:
According to the Findlib doc, this list will be topologically sorted. Don't we want some order-invariant representation here so as to be sure that we're robust w.r.t. to the precise order findlib will pick for the DAG traversal?